theatre/studio/src/store/stateEditors.ts

```diff
@@ -611,51 +616,78 @@ namespace stateEditors {
             )
           }
 
-          export function replaceKeyframes<T>(
-            p: WithoutSheetInstance<SheetObjectAddress> & {
+          export function replaceKeyframes(
+            pointers: (WithoutSheetInstance<SheetObjectAddress> & {
               trackId: string
               keyframes: Array<Keyframe>
               snappingFunction: SnappingFunction
-            },
+            })[],
```

Comment:
I've updated this to accept an array of pointers so we can update several tracks at once whilst only creating 1 undo item

Reply:
Good idea for UX, but not necessary because `studio.transaction()` can invoke multiple stateEditor fns in one transaction